src/browser/provider/built-in/dedicated/chrome/cdp-client/index.ts

```diff
@@ -232,6 +232,55 @@ export class BrowserClient {
         }
     }
 
+    public async maximizeWindow (): Promise<void> {
+        const target = await getFirstTab(this._port);
+        const client = await this.getActiveClient();
+
+        if (client) {
+            const windowParams = await client.Browser.getWindowForTarget({ targetId: target.id });
+
+            await client.Browser.setWindowBounds({ windowId: windowParams.windowId, bounds: { windowState: 'maximized' } });
+        }
+    }
+
+    public async resizeBounds (newDimensions: Size): Promise<void> {
+        const { viewportSize, config } = this._runtimeInfo;
+
+        let horizontal = 0;
+        let vertical   = 0;
+
+        if (viewportSize.outerWidth && viewportSize.outerHeight) {
+            horizontal = viewportSize.outerWidth - viewportSize.width;
```

Comment:
It's unclear what is `horizontal` and `vertical`. Let's rename to something clear.